code/modules/mob/living/basic/space_fauna/mushroom.dm

```diff
@@ -0,0 +1,192 @@
+/mob/living/basic/mushroom
+	name = "walking mushroom"
+	desc = "It's a massive mushroom... with legs?"
+	icon_state = "mushroom_color"
+	icon_living = "mushroom_color"
+	icon_dead = "mushroom_dead"
+	mob_biotypes = MOB_ORGANIC | MOB_PLANT
+
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "whacks"
+	response_harm_simple = "whack"
+
+	speed = 1
+	melee_damage_lower = 4
+	melee_damage_upper = 4
```

Comment:
```suggestion
	melee_damage_upper = 4
	maxHealth = 60
```

forgot to mention i wanted something similar for `maxHealth` but it's okay i'll fix it